pkg/daemon/daemon.go

```diff
@@ -801,99 +816,109 @@ func (dn *Daemon) getDrainLock(ctx context.Context, done chan bool) {
 	})
 }
 
-func (dn *Daemon) drainNode(name string) error {
-	glog.Info("drainNode(): Update prepared")
+func (dn *Daemon) pauseMCP() error {
+	glog.Info("pauseMCP(): check if pausing MCP is possible")
 	var err error
-	ctx, cancel := context.WithCancel(context.TODO())
-	defer cancel()
 
-	done := make(chan bool)
-	go dn.getDrainLock(ctx, done)
-	<-done
+	if utils.ClusterType != utils.ClusterTypeOpenshift {
```

Comment:
should we move (and inverse) this check to daemon.go L510 ? and only call pauseMCP in openshift cluster ?

Reply:
sure no problem